narwhal/worker/src/handlers.rs

```diff
@@ -28,6 +29,8 @@ use crate::TransactionValidator;
 #[path = "tests/handlers_tests.rs"]
 pub mod handlers_tests;
 
+pub const MAX_REQUEST_BATCHES_RESPONSE_SIZE: usize = 6_000_000;
```

Comment:
optional style suggestion, maybe put this inside the handler fn instead of at the top level of the file, so it's easier to find?